tests/testthat/test-extract.R

```diff
@@ -186,6 +186,11 @@ test_that("can pull a prepped recipe", {
     extract_recipe(workflow),
     workflow$pre$mold$blueprint$recipe
   )
+
+  expect_error(
```

Comment:
Since we don't control this error message, and since workflows uses testthat edition 3, could you please use `expect_snapshot(error = TRUE, extract_recipe(workflow, FALSE))`? That way if the ellipsis error changes (which has already happened to me before) we don't get broken on CRAN